packages/ui/src/styles/globals.css

```
@@ -293,11 +293,13 @@
   animation-range: calc(100% - 180px) calc(100% - 20px);
-}
 
-@supports not (animation-timeline: scroll()) {
-  .nav-exit {
-    animation-name: none;
+  /* `& { }` rather than a bare declaration: nested inside an at-rule that sits
+     inside a rule, a loose declaration is ambiguous per spec. */
+  @supports not (animation-timeline: scroll()) {
+    & {
+      animation-name: none;
+    }
```

Stylelint (17.14.0)

[error] Redundant nested style rule (block-no-redundant-nested-style-rules)

(block-no-redundant-nested-style-rules)